Web Inspector: give `WI.CubicBezier` a better name and move it to its own file

dcrousso opened the pull request:
#### 569bedab4430680f37add358ba9a9fe833eba5db
<pre>
Web Inspector: give `WI.CubicBezier` a better name and move it to its own file
<a href="https://bugs.webkit.org/show_bug.cgi?id=259501">https://bugs.webkit.org/show_bug.cgi?id=259501</a>

Reviewed by Timothy Hatcher.

`WI.CubicBezier` should really be named `WI.CubicBezierTimingFunction` to convey it&apos;s true purpose.

It also shouldn&apos;t really be in `Source/WebInspectorUI/UserInterface/Models/Geometry.js` since it has nothing to do with geometry.

* Source/WebInspectorUI/UserInterface/Models/CubicBezierTimingFunction.js: Added.
(WI.CubicBezierTimingFunction):
(WI.CubicBezierTimingFunction.fromCoordinates):
(WI.CubicBezierTimingFunction.fromString):
(WI.CubicBezierTimingFunction.prototype.get inPoint):
(WI.CubicBezierTimingFunction.prototype.get outPoint):
(WI.CubicBezierTimingFunction.prototype.copy):
(WI.CubicBezierTimingFunction.prototype.toString):
(WI.CubicBezierTimingFunction.prototype.solve):
(WI.CubicBezierTimingFunction.prototype._sampleCurveX):
(WI.CubicBezierTimingFunction.prototype._sampleCurveY):
(WI.CubicBezierTimingFunction.prototype._sampleCurveDerivativeX):
(WI.CubicBezierTimingFunction.prototype._solveCurveX):
* Source/WebInspectorUI/UserInterface/Models/Geometry.js:
(WI.CubicBezier): Deleted.
(WI.CubicBezier.fromCoordinates): Deleted.
(WI.CubicBezier.fromString): Deleted.
(WI.CubicBezier.prototype.get inPoint): Deleted.
(WI.CubicBezier.prototype.get outPoint): Deleted.
(WI.CubicBezier.prototype.copy): Deleted.
(WI.CubicBezier.prototype.toString): Deleted.
(WI.CubicBezier.prototype.solve): Deleted.
(WI.CubicBezier.prototype._sampleCurveX): Deleted.
(WI.CubicBezier.prototype._sampleCurveY): Deleted.
(WI.CubicBezier.prototype._sampleCurveDerivativeX): Deleted.
(WI.CubicBezier.prototype._solveCurveX): Deleted.
* Source/WebInspectorUI/UserInterface/Models/Animation.js:
(WI.Animation.prototype._updateEffect):
* Source/WebInspectorUI/UserInterface/Views/AnimationContentView.js:
(WI.AnimationContentView.prototype._refreshPreview):
* Source/WebInspectorUI/UserInterface/Views/AnimationDetailsSidebarPanel.js:
(WI.AnimationDetailsSidebarPanel.prototype._refreshEffectSection):
* Source/WebInspectorUI/UserInterface/Views/Popover.js:
(WI.Popover.prototype._animateFrame):
* Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js:
(WI.SpreadsheetStyleProperty.prototype._addTimingFunctionTokens):

* Source/WebInspectorUI/UserInterface/Views/CubicBezierTimingFunctionEditor.js: Renamed from Source/WebInspectorUI/UserInterface/Views/BezierEditor.js.
(WI.CubicBezierTimingFunctionEditor):
(WI.CubicBezierTimingFunctionEditor.createControl):
(WI.CubicBezierTimingFunctionEditor.createCoordinateInput):
(WI.CubicBezierTimingFunctionEditor.prototype.get element):
(WI.CubicBezierTimingFunctionEditor.prototype.set cubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype.get cubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype.removeListeners):
(WI.CubicBezierTimingFunctionEditor.prototype.handleEvent):
(WI.CubicBezierTimingFunctionEditor.prototype.handleKeydownEvent):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMousedown):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMousemove):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMouseup):
(WI.CubicBezierTimingFunctionEditor.prototype._updateControlPointsForMouseEvent):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCubicBezierTimingFunction.round):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCoordinateInputs):
(WI.CubicBezierTimingFunctionEditor.prototype._updateControl):
(WI.CubicBezierTimingFunctionEditor.prototype._updatePreview):
(WI.CubicBezierTimingFunctionEditor.prototype._triggerPreviewAnimation):
(WI.CubicBezierTimingFunctionEditor.prototype._resetPreviewAnimation):
(WI.CubicBezierTimingFunctionEditor.prototype._handleNumberInputInput):
(WI.CubicBezierTimingFunctionEditor.prototype._handleNumberInputKeydown):
(WI.CubicBezierTimingFunctionEditor.prototype._changeCoordinateForInput):
* Source/WebInspectorUI/UserInterface/Views/CubicBezierTimingFunctionEditor.css: Renamed from Source/WebInspectorUI/UserInterface/Views/BezierEditor.css.
(.cubic-bezier-timing-function-editor):
(.cubic-bezier-timing-function-editor &gt; .preview):
(.cubic-bezier-timing-function-editor &gt; .preview &gt; div):
(.cubic-bezier-timing-function-editor &gt; .timing):
(.cubic-bezier-timing-function-editor &gt; .preview.animate &gt; div, .cubic-bezier-timing-function-editor &gt; .timing.animate):
(@keyframes CubicBezierTimingFunctionPreview):
(.cubic-bezier-timing-function-editor &gt; .curve):
(.cubic-bezier-timing-function-editor &gt; .curve .linear):
(.cubic-bezier-timing-function-editor &gt; .curve .cubic-bezier):
(.cubic-bezier-timing-function-editor &gt; .curve .control-line):
(.cubic-bezier-timing-function-editor &gt; .curve .control-handle):
(.cubic-bezier-timing-function-editor &gt; .curve .control-handle.selected):
(.cubic-bezier-timing-function-editor &gt; .number-input-container):
(.cubic-bezier-timing-function-editor &gt; .number-input-container &gt; input):
* Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorCubicBezierTimingFunctionEditingController.js: Renamed from Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorBezierEditingController.js.
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.get initialValue):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.get cssClassName):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverWillPresent):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverDidPresent):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverDidDismiss):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype._handleCubicBezierTimingFunctionEditorCubicBezierTimingFunctionChanged):
Also rename `WI.BezierEditor` to `WI.CubicBezierTimingFunctionEditor`.

* Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js:
(WI.InlineSwatch):
(WI.InlineSwatch.prototype.didDismissPopover):
(WI.InlineSwatch.prototype._fallbackValue):
(WI.InlineSwatch.prototype._swatchElementClicked):
(WI.InlineSwatch.prototype._valueEditorValueDidChange):
* Source/WebInspectorUI/UserInterface/Views/InlineSwatch.css:
(.inline-swatch:is(.cubic-bezier-timing-function, .spring, .variable)): Renamed from `.inline-swatch:is(.bezier, .spring, .variable)`.
(.inline-swatch:is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable)): Renamed from `.inline-swatch:is(.bezier, .box-shadow, .spring, .variable)`.
(.inline-swatch:not(.read-only):is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable, .alignment):hover): Renamed from `.inline-swatch:not(.read-only):is(.bezier, .box-shadow, .spring, .variable, .alignment):hover`.
(.inline-swatch:not(.read-only):is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable, .alignment):active): Renamed from `.inline-swatch:not(.read-only):is(.bezier, .box-shadow, .spring, .variable, .alignment):active`.
(.inline-swatch:is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable) &gt; span): Renamed from `.inline-swatch:is(.bezier, .box-shadow, .spring, .variable) &gt; span`.
Also rename `WI.InlineSwatch.Type.Bezier` to `WI.InlineSwatch.Type.CubicBezierTimingFunction`.

* Source/WebInspectorUI/UserInterface/Models/TextMarker.js:
* Source/WebInspectorUI/UserInterface/Views/CodeMirrorTextMarkers.js:
(createCodeMirrorCubicBezierTimingFunctionTextMarkers): Renamed from `createCodeMirrorBezierTextMarkers`.
* Source/WebInspectorUI/UserInterface/Views/SourceCodeTextEditor.js:
(WI.SourceCodeTextEditor.prototype._updateEditableMarkers):
* Source/WebInspectorUI/UserInterface/Views/TextEditor.js:
(WI.TextEditor.prototype.createCubicBezierTimingFunctionMarkers): Renamed from `createCubicBezierMarkers`.
(WI.TextEditor.prototype.editingControllerForMarker):
Also rename `WI.TextMarker.Type.Bezier` to `WI.TextMarker.Type.CubicBezierTimingFunction`.

* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Test.html:
* Source/WebInspectorUI/.eslintrc:

* LayoutTests/inspector/animation/resources/lifecycle-utilities.js:
* LayoutTests/inspector/unit-tests/cubic-bezier-timing-function.html: Added.
* LayoutTests/inspector/unit-tests/cubic-bezier-timing-function-expected.txt: Added.
* LayoutTests/inspector/unit-tests/geometry.html:
* LayoutTests/inspector/unit-tests/geometry-expected.txt:

Canonical link: <a href="https://commits.webkit.org/266329@main">https://commits.webkit.org/266329@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4f294f48be2640b4811099ed8a6c6a15015826e2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/13557 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/13871 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/14202 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/15294 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/12894 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/16379 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/13949 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/15576 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/13724 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/14367 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/11474 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/15998 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/11655 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/12229 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/19278 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/12730 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/12397 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/15614 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/12918 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/10799 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/12186 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/12094 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/3298 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/16514 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/12761 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rcaliman-apple commented on Source/WebInspectorUI/UserInterface/Views/CubicBezierTimingFunctionEditor.js:
```diff
@@ -125,23 +125,23 @@ WI.BezierEditor = class BezierEditor extends WI.Object
+    get cubicBezierTimingFunction()
     {
-        return this._bezier;
+        return this._;
```
I think this needs to be `return this._cubicBezierTimingFunction` not `return this._`.
Probably a search-and-replace spill-over from changing the concatenation in `createCoordinateInput()`.

rcaliman-apple reviewed (approved):
Looks good to me aside from the typo fix needed in the getter for `cubicBezierTimingFunction`. Pinging @patrickangle for review / approval.

dcrousso committed:
Web Inspector: give `WI.CubicBezier` a better name and move it to its own file
https://bugs.webkit.org/show_bug.cgi?id=259501

Reviewed by Timothy Hatcher.

`WI.CubicBezier` should really be named `WI.CubicBezierTimingFunction` to convey it's true purpose.

It also shouldn't really be in `Source/WebInspectorUI/UserInterface/Models/Geometry.js` since it has nothing to do with geometry.

* Source/WebInspectorUI/UserInterface/Models/CubicBezierTimingFunction.js: Added.
(WI.CubicBezierTimingFunction):
(WI.CubicBezierTimingFunction.fromCoordinates):
(WI.CubicBezierTimingFunction.fromString):
(WI.CubicBezierTimingFunction.prototype.get inPoint):
(WI.CubicBezierTimingFunction.prototype.get outPoint):
(WI.CubicBezierTimingFunction.prototype.copy):
(WI.CubicBezierTimingFunction.prototype.toString):
(WI.CubicBezierTimingFunction.prototype.solve):
(WI.CubicBezierTimingFunction.prototype._sampleCurveX):
(WI.CubicBezierTimingFunction.prototype._sampleCurveY):
(WI.CubicBezierTimingFunction.prototype._sampleCurveDerivativeX):
(WI.CubicBezierTimingFunction.prototype._solveCurveX):
* Source/WebInspectorUI/UserInterface/Models/Geometry.js:
(WI.CubicBezier): Deleted.
(WI.CubicBezier.fromCoordinates): Deleted.
(WI.CubicBezier.fromString): Deleted.
(WI.CubicBezier.prototype.get inPoint): Deleted.
(WI.CubicBezier.prototype.get outPoint): Deleted.
(WI.CubicBezier.prototype.copy): Deleted.
(WI.CubicBezier.prototype.toString): Deleted.
(WI.CubicBezier.prototype.solve): Deleted.
(WI.CubicBezier.prototype._sampleCurveX): Deleted.
(WI.CubicBezier.prototype._sampleCurveY): Deleted.
(WI.CubicBezier.prototype._sampleCurveDerivativeX): Deleted.
(WI.CubicBezier.prototype._solveCurveX): Deleted.
* Source/WebInspectorUI/UserInterface/Models/Animation.js:
(WI.Animation.prototype._updateEffect):
* Source/WebInspectorUI/UserInterface/Views/AnimationContentView.js:
(WI.AnimationContentView.prototype._refreshPreview):
* Source/WebInspectorUI/UserInterface/Views/AnimationDetailsSidebarPanel.js:
(WI.AnimationDetailsSidebarPanel.prototype._refreshEffectSection):
* Source/WebInspectorUI/UserInterface/Views/Popover.js:
(WI.Popover.prototype._animateFrame):
* Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js:
(WI.SpreadsheetStyleProperty.prototype._addTimingFunctionTokens):

* Source/WebInspectorUI/UserInterface/Views/CubicBezierTimingFunctionEditor.js: Renamed from Source/WebInspectorUI/UserInterface/Views/BezierEditor.js.
(WI.CubicBezierTimingFunctionEditor):
(WI.CubicBezierTimingFunctionEditor.createControl):
(WI.CubicBezierTimingFunctionEditor.createCoordinateInput):
(WI.CubicBezierTimingFunctionEditor.prototype.get element):
(WI.CubicBezierTimingFunctionEditor.prototype.set cubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype.get cubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype.removeListeners):
(WI.CubicBezierTimingFunctionEditor.prototype.handleEvent):
(WI.CubicBezierTimingFunctionEditor.prototype.handleKeydownEvent):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMousedown):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMousemove):
(WI.CubicBezierTimingFunctionEditor.prototype._handleMouseup):
(WI.CubicBezierTimingFunctionEditor.prototype._updateControlPointsForMouseEvent):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCubicBezierTimingFunction.round):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCubicBezierTimingFunction):
(WI.CubicBezierTimingFunctionEditor.prototype._updateCoordinateInputs):
(WI.CubicBezierTimingFunctionEditor.prototype._updateControl):
(WI.CubicBezierTimingFunctionEditor.prototype._updatePreview):
(WI.CubicBezierTimingFunctionEditor.prototype._triggerPreviewAnimation):
(WI.CubicBezierTimingFunctionEditor.prototype._resetPreviewAnimation):
(WI.CubicBezierTimingFunctionEditor.prototype._handleNumberInputInput):
(WI.CubicBezierTimingFunctionEditor.prototype._handleNumberInputKeydown):
(WI.CubicBezierTimingFunctionEditor.prototype._changeCoordinateForInput):
* Source/WebInspectorUI/UserInterface/Views/CubicBezierTimingFunctionEditor.css: Renamed from Source/WebInspectorUI/UserInterface/Views/BezierEditor.css.
(.cubic-bezier-timing-function-editor):
(.cubic-bezier-timing-function-editor > .preview):
(.cubic-bezier-timing-function-editor > .preview > div):
(.cubic-bezier-timing-function-editor > .timing):
(.cubic-bezier-timing-function-editor > .preview.animate > div, .cubic-bezier-timing-function-editor > .timing.animate):
(@keyframes CubicBezierTimingFunctionPreview):
(.cubic-bezier-timing-function-editor > .curve):
(.cubic-bezier-timing-function-editor > .curve .linear):
(.cubic-bezier-timing-function-editor > .curve .cubic-bezier):
(.cubic-bezier-timing-function-editor > .curve .control-line):
(.cubic-bezier-timing-function-editor > .curve .control-handle):
(.cubic-bezier-timing-function-editor > .curve .control-handle.selected):
(.cubic-bezier-timing-function-editor > .number-input-container):
(.cubic-bezier-timing-function-editor > .number-input-container > input):
* Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorCubicBezierTimingFunctionEditingController.js: Renamed from Source/WebInspectorUI/UserInterface/Controllers/CodeMirrorBezierEditingController.js.
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.get initialValue):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.get cssClassName):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverWillPresent):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverDidPresent):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype.popoverDidDismiss):
(WI.CodeMirrorCubicBezierTimingFunctionEditingController.prototype._handleCubicBezierTimingFunctionEditorCubicBezierTimingFunctionChanged):
Also rename `WI.BezierEditor` to `WI.CubicBezierTimingFunctionEditor`.

* Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js:
(WI.InlineSwatch):
(WI.InlineSwatch.prototype.didDismissPopover):
(WI.InlineSwatch.prototype._fallbackValue):
(WI.InlineSwatch.prototype._swatchElementClicked):
(WI.InlineSwatch.prototype._valueEditorValueDidChange):
* Source/WebInspectorUI/UserInterface/Views/InlineSwatch.css:
(.inline-swatch:is(.cubic-bezier-timing-function, .spring, .variable)): Renamed from `.inline-swatch:is(.bezier, .spring, .variable)`.
(.inline-swatch:is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable)): Renamed from `.inline-swatch:is(.bezier, .box-shadow, .spring, .variable)`.
(.inline-swatch:not(.read-only):is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable, .alignment):hover): Renamed from `.inline-swatch:not(.read-only):is(.bezier, .box-shadow, .spring, .variable, .alignment):hover`.
(.inline-swatch:not(.read-only):is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable, .alignment):active): Renamed from `.inline-swatch:not(.read-only):is(.bezier, .box-shadow, .spring, .variable, .alignment):active`.
(.inline-swatch:is(.cubic-bezier-timing-function, .box-shadow, .spring, .variable) > span): Renamed from `.inline-swatch:is(.bezier, .box-shadow, .spring, .variable) > span`.
Also rename `WI.InlineSwatch.Type.Bezier` to `WI.InlineSwatch.Type.CubicBezierTimingFunction`.

* Source/WebInspectorUI/UserInterface/Models/TextMarker.js:
* Source/WebInspectorUI/UserInterface/Views/CodeMirrorTextMarkers.js:
(createCodeMirrorCubicBezierTimingFunctionTextMarkers): Renamed from `createCodeMirrorBezierTextMarkers`.
* Source/WebInspectorUI/UserInterface/Views/SourceCodeTextEditor.js:
(WI.SourceCodeTextEditor.prototype._updateEditableMarkers):
* Source/WebInspectorUI/UserInterface/Views/TextEditor.js:
(WI.TextEditor.prototype.createCubicBezierTimingFunctionMarkers): Renamed from `createCubicBezierMarkers`.
(WI.TextEditor.prototype.editingControllerForMarker):
Also rename `WI.TextMarker.Type.Bezier` to `WI.TextMarker.Type.CubicBezierTimingFunction`.

* Source/WebInspectorUI/UserInterface/Main.html:
* Source/WebInspectorUI/UserInterface/Test.html:
* Source/WebInspectorUI/.eslintrc:

* LayoutTests/inspector/animation/resources/lifecycle-utilities.js:
* LayoutTests/inspector/unit-tests/cubic-bezier-timing-function.html: Added.
* LayoutTests/inspector/unit-tests/cubic-bezier-timing-function-expected.txt: Added.
* LayoutTests/inspector/unit-tests/geometry.html:
* LayoutTests/inspector/unit-tests/geometry-expected.txt:

Canonical link: https://commits.webkit.org/266329@main
